modules/express/src/clientRoutes.ts

```diff
@@ -352,6 +352,15 @@ function handleCanonicalAddress(req: express.Request) {
   return (coin as Coin.Bch | Coin.Bsv | Coin.Ltc).canonicalAddress(address, version || fallbackVersion);
 }
 
+function handleV1Sign(req: express.Request) {
+  console.log('v1 sign was called');
+  return 'v1_sign_transaction';
+}
```

Comment:
let's change these to just throw an error for now instead
```suggestion
  throw new Error('not yet implemented');
}
```

Reply:
done